index.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   name: 'ember-cli-rails-addon',
 
   init: function() {
+    this._super.apply(this, arguments);
```

Comment:
Can you update this to the snippet listed in the deprecation warning? As written here, it is only compatible with CoreObject@2, and will break on earlier versions of ember-cli.